app/frontend/.env.development

```diff
@@ -1,5 +1,6 @@
 REACT_APP_COUCHERS_ENV=dev
 REACT_APP_API_BASE_URL=http://localhost:8888
+REACT_APP_MEDIA_BASE_URL="http://127.0.0.1:5000"
```

Comment:
Is it because you're testing on `localhost` and your browser counts `127.0.0.1` as a separte origin? For CORS

Reply:
possibly